### PR TITLE
Roller Skates, New Traitor shoes

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

New traitor item for 3 tc to speed you up by 45%!

## Why It's Good For The Game

Adds a way for traitors to have a better ways to get away from crime scenes that can be removed with bola's rather then brokenly powerful no slowdown stuff

## Changelog
:cl:
add: Traitors have been seen with riding around in Roller Skates... 
/:cl:

### CODE REVIEW

**Comment by Arturlang** [comment]
You do realize how powerfull being fast in this game is? Also we are trying to reduce the amount of ways to go sanic speed as its not a lot of fun anyways. And this item is horrifyingly cheap for what it does to speed you up so fucking much

**Comment by Arturlang** [comment]
Hobestly i dont think shoes that bearly double your speed at bearly no downside for only 3 tc is a good idea

**Comment by ghost** [comment]
blessed

**Comment by silicons** [comment]
🤔

**Comment by B1gb3ast** [comment]
I mean.
If the speedboost doesn't stack with sprint, its fine.
No stamin for a speed inbetweeen sprint and run: that's k.
50% faster sprints: bad.

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1359,6 +1359,13 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 	cost = 2
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
+/datum/uplink_item/suits/roller_skates
+	name = "Roller Skates"
+	desc = "Shoes with hidden roller blades to speed away from the crime. Note - Roller blades are not hidden"
+	item = /obj/item/clothing/shoes/rollers
+	cost = 3
+	exclude_modes = list(/datum/game_mode/nuclear) //Nukies dont get cool things
```
Comment (B1gb3ast) [inline code/modules/uplink/uplink_items.dm:1367]: What the fuck trillby I want 5 terrorists on rollerskates and I want them now.

Comment (Trilbyspaceclone) [reply code/modules/uplink/uplink_items.dm:1367]: They already get so much in terms of speed, and making them even faster just dosnt seem all that good. Also 45% not 50% speed buff. If its to brokenly good ill make it 20%

**Comment by silicons** [comment]
sorry, this seems like a fun item but at the same time i don't think i want to have another stack-the-speedup game.

**Comment by Trilbyspaceclone** [comment]
Sure not problem.

**Pull request closed without merging**